### PR TITLE
Terminology formatting updates

### PR DESCRIPTION
**Pull request by sheriff-rh**
Terminology style updates for the Registry book.
https://termformatupdates--ocpdocs.netlify.app/openshift-enterprise/latest/registry/architecture-component-imageregistry.html

### CODE REVIEW

**Review by pneedle-rh — COMMENTED** [review]
@sheriff-rh I have now completed my review.

In addition to the line-specific comments, please review the following comments:

- There are a few references to "ConfigMap` in [Image [config.openshift.io/v1]](https://termformatupdates--ocpdocs.netlify.app/openshift-enterprise/latest/rest_api/config_apis/image-config-openshift-io-v1.html) and in the Registry book in general. Should these either be in back ticks and stated as objects (e.g. "`ConfigMap` object"), or "config map"?

- "openshift-config" is referenced in the same assembly as a namespace, but it is not in back ticks.

- There are references to "ImageStreamMappings" in the same assembly and in the Registry book. You could change those to "image stream mappings" if they are general references, or put them in back ticks and state that they are an object, if they are specific references to objects. The same goes for references to "ImageStreams".

- The **Description** sections throughout the same assembly have reference to properties such as "additionalTrustedCA" that are not in back ticks. If they are direct YAML property references then should these be in back ticks?

- The same assembly has a reference to "builds+pods" that is not in back ticks. Is this a reference to a property that should be in back ticks, or should it be "builds and pods"?

- There is a note in [Image Registry on bare metal and vSphere](https://termformatupdates--ocpdocs.netlify.app/openshift-enterprise/latest/registry/configuring-registry-operator.html#image-registry-on-bare-metal-vsphere) that states ""Image Registry has been removed. `ImageStreamTags`, `BuildConfigs` and `DeploymentConfigs` which reference `ImageStreamTags` may not work as expected." Should those be lower case, word separated general references? If not, should they be singular rather than plural, as well as having "object" or suchlike stated afterwards? The same note is repeated elsewhere in the book.

- There are a few references to "Operator" that are not referencing a full pronoun that are capitalised (for example "...credentials used by the Operator"). Since these are general references to operators, but the specific name of the operator is not being stated, should these be lowercase? Examples of this can be found in [Image Registry Operator configuration parameters](https://termformatupdates--ocpdocs.netlify.app/openshift-enterprise/latest/registry/configuring-registry-operator.html#registry-operator-configuration-resource-overview_configuring-registry-operator), for instance.

- "configs.imageregistry.operator.openshift.io" is referenced in [Image registry removed during installation](https://termformatupdates--ocpdocs.netlify.app/openshift-enterprise/latest/registry/configuring_registry_storage/configuring-registry-storage-baremetal.html#registry-removed_configuring-registry-storage-baremetal) but it is not in back ticks.

**Comment by pneedle-rh** [comment]
@sheriff-rh since my peer review, I have discussed whether "Operator" should be capitalised when making general references (i.e. when not stating a specific operator name) with @bergerhoffer . I now understand that Operator should be capitalised and that it is an exception. So, please ignore my comments in this PR about potentially setting "Operator" to lowercase in those general instances.

File: registry/configuring-registry-operator.adoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ configuration detected in the cluster.
 
 If insufficient information is available to define a complete
 `configs.imageregistry.operator.openshift.io` resource, the incomplete resource
-will be defined and the operator will update the resource status with
+will be defined and the Operator will update the resource status with
```
[the remark this answers is not in the document]
Comment (sheriff-rh) [reply registry/configuring-registry-operator.adoc:28]: Will be leaving this as-is per notes on Slack. Thanks!

File: registry/configuring-registry-operator.adoc [hunk 2]
```diff
@@ -35,7 +35,7 @@ and workload resources for the registry reside in that namespace.
 
 [IMPORTANT]
 ====
-The Image Registry Operator's behavior for managing the pruner is orthogonal to the `ManagementState` specified on the Image Registry Operator's `ClusterOperator` object. If the image registry operator is not in the `Managed` state, the image pruner can still be configured and managed by the Pruning Custom Resource.
+The Image Registry Operator's behavior for managing the pruner is orthogonal to the `ManagementState` specified on the Image Registry Operator's `ClusterOperator` object. If the Image Registry Operator is not in the `Managed` state, the image pruner can still be configured and managed by the Pruning Custom Resource.
```
[the remark this answers is not in the document]
Comment (sheriff-rh) [reply registry/configuring-registry-operator.adoc:38]: Decided to go with `Pruning` custom resource. Good catch!

File: registry/configuring-registry-operator.adoc [hunk 3]
```diff
@@ -35,7 +35,7 @@ and workload resources for the registry reside in that namespace.
...
 
 However, the `ManagementState` of the Image Registry Operator alters the behavior of the deployed image pruner job:
```
[the remark this answers is not in the document]
Comment (sheriff-rh) [reply registry/configuring-registry-operator.adoc:40]: Yep - this should be `managementState`. Fixing in all locations in this book.

File: rest_api/config_apis/image-config-openshift-io-v1.adoc [hunk 4]
```diff
@@ -126,16 +126,16 @@ Type::
...
 | `insecure`
 | `boolean`
-| insecure indicates whether the registry is secure (https) or insecure (http) By default (if not specified) the registry is assumed as secure.
+| `insecure` indicates whether the registry is secure (https) or insecure (http) By default (if not specified) the registry is assumed as secure.
 
 |===
 ..spec.registrySources
 Description::
-  registrySources contains configuration that determines how the container runtime should treat individual registries when accessing images for builds+pods. (e.g. whether or not to allow insecure access).  It does not contain configuration for the internal cluster registry.
+  `registrySources` contains configuration that determines how the container runtime should treat individual registries when accessing images for builds+pods. (e.g. whether or not to allow insecure access).  It does not contain configuration for the internal cluster registry.
```
Comment (pneedle-rh) [inline rest_api/config_apis/image-config-openshift-io-v1.adoc:138]: Should "builds+pods" be "builds and pods"?

File: rest_api/config_apis/image-config-openshift-io-v1.adoc [hunk 5]
```diff
@@ -178,11 +178,11 @@ Type::
 
 | `externalRegistryHostnames`
 | `array (string)`
-| externalRegistryHostnames provides the hostnames for the default external image registry. The external hostname should be set only when the image registry is exposed externally. The first value is used in 'publicDockerImageRepository' field in ImageStreams. The value must be in "hostname[:port]" format.
+| `externalRegistryHostnames` provides the hostnames for the default external image registry. The external hostname should be set only when the image registry is exposed externally. The first value is used in `publicDockerImageRepository` field in image streams. The value must be in `hostname[:port]`` format.
```
Comment (pneedle-rh) [inline rest_api/config_apis/image-config-openshift-io-v1.adoc:181]: You have two back ticks after "hostname[:port]". Should this be only one back tick?

**Review by pneedle-rh — COMMENTED** [review]
@sheriff-rh thank you for your updates!

The line specific comments have been resolved, but the additional comments that I made in [this update](https://github.com/openshift/openshift-docs/pull/27492#pullrequestreview-535293139) have not been applied. Can you please review those? Thanks!

I have also added a couple of additional comments below.

File: rest_api/config_apis/image-config-openshift-io-v1.adoc (hunk 5 above)
Comment (sheriff-rh) [reply rest_api/config_apis/image-config-openshift-io-v1.adoc:181]: Yep, good catch, thank you!

File: rest_api/config_apis/image-config-openshift-io-v1.adoc (hunk 4 above)
Comment (sheriff-rh) [reply rest_api/config_apis/image-config-openshift-io-v1.adoc:138]: Yep, not sure how I missed this twice when you called it out previously. Fixed.

**Comment by pneedle-rh** [comment]
LGTM!

**Pull request merged**